content/docs/learning-guide/tutorials/ai/kythera-city-level-overview.md

```diff
@@ -0,0 +1,219 @@
+---
+linkTitle: Kythera City Level Overview
+title: Kythera City Level Overview
+description: Overview of the AI demo level "Kythera City"
+weight: 100
+---
+
+![A ship navigates Kythera City in 3D](/images/learning-guide/tutorials/ai/kythera-city-ship.png)
+
+The `KytheraCity` level demonstrates simple examples of setup for several Kythera AI features, in the Open3D (developers preview) build of the Kythera AI gem.
+
+The Kythera AI gem features demonstrated in this level are:
```

Comment:
```suggestion
The `KytheraCity` level demonstrates basic setup examples for several features of the **Kythera AI** Gem in **Open 3D Engine (O3DE)** [Developer Preview]. These features include:
```
A few recommended edits:
* To minimize redundancy, combine "Kythera AI features" and "the Kythera AI Gem" into a single reference to "Kythera AI."
* Per our [terminology guidelines](https://docs.o3de.org/docs/contributing/to-docs/terminology/):
  * "Use 'Open 3D Engine (O3DE)' in bold for the first on-page reference."
    * For consistency with the [Download page](https://o3de.org/download/), use "[Developer Preview]" for this release.
  * "'Gem' is a proper noun and must be capitalized."
  * Minor edits for flow. Consider using "basic" instead of "simple", which can feel condescending to a reader who might find it difficult.
* For brevity, combine the features list intro sentence with the first sentence, then shorten it, as in the suggested rewrite.